app/angular/src/server/framework-preset-angular.ts

```diff
@@ -47,19 +41,45 @@ export function webpack(
             { loader: 'sass-loader' },
           ],
         },
+        //   {
+        //     test: /\.tsx?$/,
+        //     use: [
+        //       {
+        //         loader: 'ts-loader',
+        //         options: tsLoaderOptions
+        //       },
+        //       { loader: path.resolve(__dirname, 'ngx-template-loader') }
+        //     ]
+        //   },
+        {
+          test: /[/\\]@angular[/\\]core[/\\].+\.js$/,
+          parser: { system: true },
+        },
+        //   {
+        //     test: /\.html$/,
+        //     loader: 'raw-loader',
+        //     exclude: /\.async\.html$/
+        //   },
       ],
     },
     resolve: {
       ...config.resolve,
     },
     plugins: [
       ...config.plugins,
+      new AngularCompilerPlugin({
+        tsConfigPath: tsLoaderOptions.configFile,
+        mainPath: path.join(__dirname, '../client/preview/angular/helpers.js'),
```

Comment:
@stupidawesome 
This was a long shot. I'm not sure how to point to the file that executes `bootstrapModule` 
You said in order to get JIT working this fn call can be in a file with lots of other stuff, right? 
Still, it's hidden in a function call and I guess I'm doing it wrong 🙂